### ISSUE TITLE
Error: Error calculating min/max zoom: Bounds invalid

### ISSUE THREAD

**Issue by NickHeiner** [mapbox/mapbox-studio-classic#1447]
I am using Mapbox Studio `0.2.7`. I am seeing an error, but I'm not sure how to fix it.
### Repro steps
1. Launch Mapbox studio
2. Click "projects"
3. Create a new project
4. Select "Blank source"
5. Click "+ New layer"
6. Choose a `geojson` file that contains the following:

``` json
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "geometry": {
        "type": "Polygon",
        "coordinates": [
          [
            [
              -0.004,
              -0.004
            ],
            [
              0.004,
              -0.004
            ],
            [
              0.004,
              0.004
            ],
            [
              -0.004,
              0.004
            ],
            [
              -0.004,
              -0.004
            ]
          ]
        ]
      },
      "properties": {}
    }
  ]
}
```

**Expected**: I see my feature in the source viewer.
**Actual**: I get the following error:

> Error: Error calculating min/max zoom: Bounds invalid

![mapbox error](https://cloud.githubusercontent.com/assets/829827/9662809/81991274-5261-11e5-9c87-835fe6678108.png)

I may be doing something wrong, but if this is user error, the error message is not helpful in explaining what's wrong with my data. The same data works fine when I paste it in to geojson.io.

Thanks for your help!

**Comment by wilhelmberg** [comment]
@NickHeiner I can replicate with the latest dev build of Mapbox Studio.
Looking into it ....

**Comment by NickHeiner** [comment]
Awesome. Thanks.

**Comment by springmeyer** [comment]
@BergWerkGIS is this still replicable? Is it due to https://github.com/mapbox/mapnik-omnivore/issues/88?

Note: removed this from the v0.2.8 milestone since that release has already flown.

**Comment by wilhelmberg** [comment]
@springmeyer yes, due to https://github.com/mapbox/mapnik-omnivore/issues/88 but originates from `sphericalmercator`.

Haven't checked yet, but might have been solved by https://github.com/mapbox/node-sphericalmercator/commit/99b17908aba7863f03e60a9143e789c30408d54f?diff=split
